src/server/context.spec.ts

```diff
@@ -19,74 +22,135 @@ describe('context', () => {
       givenUrl: 'dont-care.com',
     },
   };
-  let batchUrlFnSpy;
-  let batchIdFnSpy;
-  let context: IContext;
-
-  beforeEach(() => {
-    batchUrlFnSpy =
-      SavedItemDataService.prototype.batchGetSavedItemsByGivenUrls = jest
-        .fn()
-        .mockResolvedValue([savedItem]);
-    batchIdFnSpy = SavedItemDataService.prototype.batchGetSavedItemsByGivenIds =
-      jest.fn().mockResolvedValue([savedItem]);
-    context = new ContextManager({
+  describe('event emitter', () => {
+    let sentryEventSpy;
+    let sentryExceptionSpy;
+    const context = new ContextManager({
       request: {
         headers: { userid: '1', apiid: '0' },
       },
       dbClient: jest.fn() as unknown as Knex,
-      eventEmitter: null,
+      eventEmitter: new ItemsEventEmitter(),
+    });
+    beforeEach(() => {
+      sinon.restore();
+      sentryEventSpy = sinon.spy(Sentry, 'captureEvent');
+      sentryExceptionSpy = sinon.spy(Sentry, 'captureException');
+    });
+    afterAll(() => sinon.restore());
+    it('should log a warning to Sentry if save is undefined', async () => {
+      await context.emitItemEvent(EventType.ARCHIVE_ITEM, undefined);
+      expect(sentryEventSpy.callCount).to.equal(1);
+      const event = sentryEventSpy.getCall(0).args[0];
```

Comment:
is `getCall(0).args[0]` just fetching us the arguments used to make the first call with this spy object?

Reply:
yep!